ugs-platform/pom.xml

```diff
@@ -31,14 +31,12 @@
       </ugs.modules.specification.version>
 
       <!--==== JRE Bundle Properties ====-->
-      <ugs.bundle.java.url>http://download.oracle.com/otn-pub/java/jdk/8u171-b11/512cd62ec5174c3487ac17c61aaa89e8</ugs.bundle.java.url>
+      <ugs.bundle.java.url>https://download.java.net/java/GA/jdk12.0.2/e482c34c86bd4bf8b56c0b35558996b9/10/GPL/</ugs.bundle.java.url>
```

Comment:
What do you think about switching to adopt open jdk?

https://github.com/AdoptOpenJDK/openjdk12-binaries/releases

Reply:
I'm supposed to be a Java-developer and I've never heard of this? =)
Looks nice, I'll look into it!

Reply:
It worked fine with the adoptopenjdk:s builds. They also has ARM-binaries which works with raspberry pi. Good stuff!